checkpoint/client/cli/tx.go

```diff
@@ -45,6 +47,35 @@ func SendCheckpointTx(cdc *codec.Codec) *cobra.Command {
 		Short: "send checkpoint to tendermint and ethereum chain ",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			if viper.GetBool(FlagAutoConfigure) {
+				isProposerBytes, _, err := cliCtx.Query(fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryProposer))
```

Comment:
It would be great if we can use `staking/current-proposer` instead of new proposer querier.

Reply:
I knew there is a querier already, didn't want to pass that here. Will create new obj now from staking keeper.

Reply:
Yeah that would be the best, missed that.